### PR TITLE
Disable Mash warnings

### PR DESCRIPTION
**Pull request by hnmx4**
for https://github.com/itamae-kitchen/itamae/issues/243.
I use a Hashie developer’s solution that solves issue similar to issue#243 as  reference.

https://github.com/intridea/hashie/issues/394
> I just released Hashie v3.5.2 which has the disable_warnings feature for the Mash logger and sets the Hashie logger to the Rails logger when we detect that we're running in Rails. Hopefully, this helps!

https://github.com/intridea/hashie
> You can also disable the logging in subclasses of Mash:
>
> ```
> class Response < Hashie::Mash
>   disable_warnings
> end
> ```

### CODE REVIEW

**Comment by hnmx4** [comment]
@unasuke hi. Congratulations on appointment to Itamae committee :tada: . I merged the master branch to this branch. Could you merge this branch?

File: lib/itamae/node.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 require 'itamae'
+require 'itamae/mash'
```
Comment (sue445) [inline lib/itamae/node.rb]: I prefer add `require 'itamae/mash'` to [lib/itamae.rb](https://github.com/itamae-kitchen/itamae/blob/v1.9.11/lib/itamae.rb) than here.

@unasuke Do you think?

**Comment by unasuke** [comment]
@hnmx4 Thank you for catch-up! But still printing warn message on CI. :cry:  Could you fix it?

https://travis-ci.org/itamae-kitchen/itamae/jobs/439940863

File: lib/itamae/node.rb (hunk 1 above)
Comment (unasuke) [reply lib/itamae/node.rb]: I agree!

**Comment by hnmx4** [comment]
@unasuke Thank you for reply. Fix later. 🙃

**Comment by sue445** [comment]
@hnmx4 ping

**Comment by unasuke** [comment]
It merged in #273 . Thank you @hnmx4 ! :heart_eyes:

**Pull request closed without merging**